pkg/cmd/root/help_topic.go

```diff
@@ -6,6 +6,25 @@ import (
 )
 
 var HelpTopics = map[string]map[string]string{
+	"mintty": {
+		"short": "Information about using gh with MinTTY",
+		"long": heredoc.Doc(`
+			MinTTY is the terminal emulator that comes by default with Git
+			for Windows.  It has known issues with gh's ability to prompt a
+			user for input.
+
+			There are a few workarounds to make gh work with MinTTY:
+
+			- Reinstall Git for Windows. 
+			  Be sure to check the box "Enable experimental support for pseudo consoles".
+
+			- Use a different terminal emulator with Git for Windows like Windows Terminal or cmd.exe.
+			  Try running "C:\Program Files\Git\bin\bash.exe" in one of those other terminal emulators.
```

Comment:
Hmm, is this really the only way to consistently open Git Bash from Windows Terminal? Isn't there some way to create a shortcut of some kind so that people don't have to type or paste the full path?

Reply:
You can set up a Windows Terminal profile to always run bash.exe or set up a shortcut, but both of those strategies seemed out of scope for this help topic.